spec/active_yaml/base_spec.rb

```diff
@@ -20,13 +20,26 @@ class Empty        < ActiveYaml::Base ; end # Empty YAML
     Object.send :remove_const, :ArrayRow
     Object.send :remove_const, :City
     Object.send :remove_const, :State
+    Object.send :remove_const, :User
     Object.send :remove_const, :Empty
   end
 
   describe ".load_path" do
-    it 'can execute embedded ruby' do
-       expect(User.first.email).to match(/^user[0-9]*@email.com$/)
-       expect(User.first.password).to eq('secret')
+    context 'default' do
+      it 'can execute embedded ruby' do
+        User.first.email.should =~ /^user[0-9]*@email.com$/
+        User.first.password.should == 'secret'
```

Comment:
we are now using the expect syntax.

```suggestion
        expect(User.first.email).to match /^user[0-9]*@email.com$/
        expect(User.first.password).to eq 'secret'
```